configs/model/lr_scheduler/ReduceLROnPlateau.yaml

```diff
@@ -0,0 +1,10 @@
+scheduler:
+  _target_: torch.optim.lr_scheduler.ReduceLROnPlateau
+  _partial_: true
+  mode: min
+  factor: 0.1
+  patience: 10
+config:
+  monitor: ${data.monitor_split}/loss
+  interval: epoch
+  frequency: 1
```

Comment:
Just a detail but I am questioning the clarity of putting the lr_schedulers in `configs/model/lr_scheduler` ?

Reply:
The idea was to refactor them away, since most models will use the same general parameters for the schedulers and optimizers, and then if the models need to configure them further (ex. setting the learning rate), they can simply override those particular parameters.

This also should make it easier to swap out different schedulers and optimizers, in a perfect world. Although this probably isn't the best solution. Ideally I would like to be able to easily swap out the optimizers and LR schedulers without worrying about the model itself.  If you have any ideas on how to structure this please let me know, I am struggling a bit with the config structure.